Sprint-1/exercises/paths.js

```diff
@@ -16,3 +16,7 @@ console.log(`The base part of ${filePath} is ${base}`);
 
 // Create a variable to store the dir part of the filePath variable
 // Create a variable to store the ext part of the variable
+let dirPath = filePath.slice(0, lastSlashIndex);
+let extPart = base.slice(-4);
```

Comment:
Does this line still work if the extension was `.md` instead? Can you think of a way to find the file's extension with the use of `.`? As this will divide your file name and extension.

And a minor point: I don't consider `.` to be part of the extension, so for example, the extension here is `txt`.